### PR TITLE
Fixed typos and added link to part 1

### PR DESCRIPTION
**Pull request by hessio**
Signed-off-by: Patrick Hession <patrick.hession@seagate.com>

### Changes
 - Make sure users know what the purpose of this guide
 - Make sure users get the correct commands

### How Has This Been Tested? (Optional)
 - VMWare Workstation 16 Pro
 

-----
[View rendered doc/ProvisionReleaseBuild.md](https://github.com/Seagate/cortx/blob/hessio-patch-3/doc/ProvisionReleaseBuild.md)

### CODE REVIEW

File: doc/ProvisionReleaseBuild.md [hunk 1]
```diff
@@ -12,31 +15,32 @@
    ``` CORTX_RELEASE_REPO="file:///var/artifacts/0"```
    
    - Install Provisioner API and requisite packages
-   ```yum install -y yum-utils
-      yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
-      yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/cortx_iso/"
+   ```
+   yum install -y yum-utils
```
Comment (mukul-seagate11) [inline doc/ProvisionReleaseBuild.md:19]: please use "#" before the commands (all the below commands) after sudo is done

File: doc/ProvisionReleaseBuild.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Provision Release Build
 
+## Prerequisite
+Note: This guide is intended to run the build release created in a previous [guide](https://github.com/Seagate/cortx/blob/main/doc/Release_Build_Creation.rst). Please complete this [guide](https://github.com/Seagate/cortx/blob/main/doc/Release_Build_Creation.rst) before following this guide.
+
 ### 1.  Change Hostname
    ```sudo hostnamectl set-hostname deploy-test.cortx.com```
```
Comment (mukul-seagate11) [inline doc/ProvisionReleaseBuild.md:7]: please use "$" before the commands

**Comment by mukul-seagate11** [comment]
@ypise, can you review/comment from Provisioner point of view?

**Comment by ypise** [comment]
Looks good to me.
Thanks @hessio for the help.

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Pull request merged**